src/components/Footer/Category/Category.js

```diff
@@ -1,11 +1,11 @@
-import React, { Fragment } from 'react';
-
+import React, { memo } from 'react';
 const category = (props) => (
-  <Fragment>
-    <li>All</li>
-    <li>Active</li>
-    <li>Completed</li>
-  </Fragment>
+  <li>
+  <a
+    className={`${props.category === props.name.toUpperCase() ? "selected" : ''}`}
```

Comment:
check condition in `props.name` is null, `props.name.toUpperCase()` will be error